site/labs/objects/collections.rst

```diff
@@ -0,0 +1,76 @@
+***********
+Collections
+***********
```

Comment:
How much changed here compared to the old files?

Reply:
Literally all new.

Reply:
I see -- two PRs would have been nice.

Reply:
Ack